package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "1pr",
+  "version": "1.0.0",
+  "description": "I will be accepting up to one pull request per day on this project.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/thepracticaldev/1pr.git"
+  },
+  "keywords": [],
+  "author": "",
```

Comment:
You're right, I'll remove those two properties. They're generated by default through `npm init -y`.